### PR TITLE
LINT: Fixup black string normalization

### PR DESCRIPTION
**Pull request by jcrist**
After running black, several places in our codebase were rewritten from
something like

```
raise ValueError("part one of the message "
                 "part two")
```

to

```
raise ValueError("part one of the message " "part two")
```

This fixes those cases, removing the unnecessary two-part string.

Stumbled across this when doing some other work, and it was a
quick fix. See also https://github.com/dask/dask/pull/5227.

### CODE REVIEW

File: distributed/cli/dask_scheduler.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def main(
     tls_cert,
     tls_key,
     dashboard_address,
-    **kwargs
+    **kwargs,
```
Comment (mrocklin) [inline distributed/cli/dask_scheduler.py]: This excludes Python 3.5 .  You'll have to update your black to master, or the commit in the pre-commit config file.   :/

Comment (jcrist) [reply distributed/cli/dask_scheduler.py]: Ah, didn't realize this required a non-released version of black.

Comment (jcrist) [reply distributed/cli/dask_scheduler.py]: Found the lines that were confusing black and fixed them, should now work fine with the latest black release or master.

**Comment by mrocklin** [comment]
Thanks @jcrist !

**Pull request merged**